doc/classes/ConfigFile.xml

```diff
@@ -161,61 +164,70 @@
 			<returns_error number="0"/>
 			<returns_error number="12"/>
 			<param index="0" name="path" type="String" />
+			<param index="1" name="allow_objects" type="bool" default="false" />
 			<description>
-				Loads the config file specified as a parameter. The file's contents are parsed and loaded in the [ConfigFile] object which the method was called on.
+				Loads the config file specified as a parameter. The file's contents are parsed and loaded in the [ConfigFile] object which the method was called on. If [param allow_objects] is [code]true[/code], decoding objects is allowed.
 				Returns [constant OK] on success, or one of the other [enum Error] values if the operation failed.
+				Internally, this uses the same decoding mechanism as the [method @GlobalScope.str_to_var] and [method @GlobalScope.str_to_var_with_objects] methods.
+				[b]Warning:[/b] Deserialized objects can contain code which gets executed. Do not use this option if the serialized object comes from untrusted sources to avoid potential security threats such as remote code execution.
 			</description>
 		</method>
 		<method name="load_encrypted">
 			<return type="int" enum="Error" />
 			<param index="0" name="path" type="String" />
 			<param index="1" name="key" type="PackedByteArray" />
+			<param index="2" name="allow_objects" type="bool" default="false" />
 			<description>
-				Loads the encrypted config file specified as a parameter, using the provided [param key] to decrypt it. The file's contents are parsed and loaded in the [ConfigFile] object which the method was called on.
-				Returns [constant OK] on success, or one of the other [enum Error] values if the operation failed.
+				Loads the encrypted config file specified as a parameter, using the provided [param key] to decrypt it. The file's contents are parsed and loaded in the [ConfigFile] object which the method was called on. See [method load] for details.
 			</description>
 		</method>
 		<method name="load_encrypted_pass">
 			<return type="int" enum="Error" />
 			<param index="0" name="path" type="String" />
 			<param index="1" name="password" type="String" />
+			<param index="2" name="allow_objects" type="bool" default="false" />
 			<description>
-				Loads the encrypted config file specified as a parameter, using the provided [param password] to decrypt it. The file's contents are parsed and loaded in the [ConfigFile] object which the method was called on.
-				Returns [constant OK] on success, or one of the other [enum Error] values if the operation failed.
```

Comment:
Why removing this? `parse` and `save` still mention it. Either all or nothing(?)

Reply:
These methods are located next to each other (unlike `parse()`), have similar names and signatures. In my opinion, it's a bit verbose to have 3 methods in a row with identical 3 paragraphs. In some cases we duplicate parts, in others we use "See x for details".

The same with `save*()`. I made a full description only for `save()`, for `save_encrypted()` and `save_encrypted_pass()` I left only unique information and added "See `save()` for details".

Reply:
Then the connection for the details needs to be made a bit more obvious, as it's a bit difficult to read at a glance. Perhaps putting _"See save() for details"_ onto a new line?

Reply:
Personally I would prefer a new line too, but in a similar case I was asked to keep it on one line if I remember correctly.

Reply:
`\n\t+See \[method.*\] for details` - 31 matchs
doc/classes/ConfigFile.xml: 4
doc/classes/Control.xml: 14
doc/classes/Window.xml: 13

` See \[method.*\] for details` - 17 matchs
doc/classes/CollisionObject2D.xml: 1
doc/classes/ConvexPolygonShape2D.xml: 1
doc/classes/DisplayServer.xml: 4
doc/classes/FileAccess.xml: 4
doc/classes/PopupMenu.xml: 3
doc/classes/RichTextLabel.xml: 1
doc/classes/SceneTree.xml: 1
doc/classes/SkeletonModification2DCCDIK.xml: 1
doc/classes/TabContainer.xml: 1